tsconfig.json

```diff
@@ -1,28 +1,28 @@
 {
```

Comment:
This was a result of running `yarn lint-fix` manually on one of my commits, which processed all JSON files in the repo to get them in line with our `prettier` config :-)